CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ option(SSG_SVG_IN_XCCDF_ENABLED "If enabled, the built XCCDFs will include the S
 option(SSG_SEPARATE_SCAP_FILES_ENABLED "If enabled, separate SCAP files (OVAL, XCCDF, CPE dict, ...) will be installed alongside the source data-streams" TRUE)
 option(SSG_ANSIBLE_PLAYBOOKS_ENABLED "If enabled, Ansible Playbooks for each profile will be built and installed." TRUE)
 option(SSG_BASH_SCRIPTS_ENABLED "If enabled, Bash remediation scripts for each profile will be built and installed." TRUE)
+option(SSG_ANSIBLE_TASKS_ENABLED "If enabled, Ansible Tasks for each rule will be installed." FALSE)
```

Comment:
What about guarding the call of the ssg_build_ansible_playbooks macro by this option?

Reply:
it makes sense

Reply:
can you please check if this is what you have in mind? https://github.com/ComplianceAsCode/content/pull/7039/commits/1ec2353dc17c2c97e2c1a4ee66960f8723372afb